projects/packages/forms/src/blocks/contact-form/components/akismet-panel/index.js

```diff
@@ -0,0 +1,57 @@
+import { Button } from '@wordpress/components';
+import { store as coreStore } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import PluginIntegrationPanel from '../shared/plugin-integration-panel';
+import './styles.css';
+
+const AkismetPanel = () => {
+	const { adminUrl } = useSelect( select => {
+		const site = select( coreStore ).getEntityRecord( 'root', 'site' );
+		return {
+			adminUrl: site?.url ? `${ site.url }/wp-admin/` : null,
+		};
+	}, [] );
```

Comment:
For getting the admin url, is this approach reliable, or would it be better to get the url via PHP and pass it to our JS via wp_localize_script()?

Reply:
There's a helper hook somewhere, let's use that.

Users can have sub-folder installs of WP, multi-site installs, etc

Reply:
I'm looking at adopting this method. For some reason, that method is not actually exported from the @automattic/jetpack-components at the moment, so checking on Slack about best approach.

Reply:
Based on feedback on Slack, I've updated to use the [getAdminUrl](https://github.com/Automattic/jetpack/blob/8e30f68bad9351aafefec7d545fb9ee62d8c84d0/projects/js-packages/script-data/src/utils.ts#L28) method from the script-data package.

Reply:
Follow up: I actually changed this again. We also had to solve the problem of the admin url being different for classic vs modern views. I've fixed that by adding a method to get the right admin url in php. See related [comment](https://github.com/Automattic/jetpack/pull/41826#discussion_r1970834244).